src/scrape_up/leetcode/__init__.py

```diff
@@ -0,0 +1,13 @@
+from leetcode_scraper import LeetCodeScraper
+
+# Create an instance of the LeetCodeScraper
+scraper = LeetCodeScraper()
+
```

Comment:
Hey @Hansika002, the `__init__.py` file is not well scripted. Look for other classes `__init__.py file for reference.